### DISCUSSION TITLE
Changing language of bot's answers

### DISCUSSION THREAD

**Discussion by Orion-89** [jagrosh/MusicBot#670] [Questions] [answered]
Hi! I don't know how to code much, but I assume that changing the bot's messages is simple. Can you give the cue where the general phrases written (I mean which file)? Thank you.

**Comment by jagrosh** [comment, marked as answer]
Editing the source is not supported.

**Reply by GuardedBroD** [reply]
Sorry for necroing if thats even a thing on github im not sure im new to this but will there ever be a way to change the source code even just the responses/commands to change it thatd be great to have

**Reply by Lastorder-DC** [reply]
Basically you have to change almost all `event.reply`, `event.replyInDm`, `event.replyError` calls. (`this.help` too if you want to change command help too)

Of course, modified code will not supported by developer. (and editing is also not supported - see README.md for details about editing source code) but if want to do that, you can see [my commits](https://github.com/Lastorder-DC/MusicBotKR/commits/ko-kr) for changing language for bot.(some changes like `JosaFormatter` are for handing korean language, thus can be ignored)

@GuardedBroD

**Reply by GuardedBroD** [reply]
Sorry for bothering you I'm very new to java let alone coding in general but whenever I change event.reply calls to a different response it does not change when I actually work the bot and use the command I changed the response to. Is there anything I need to do beforehand or anything else needed to be changed before it will work?

**Reply by Lastorder-DC** [reply]
@GuardedBroD You should compile your code(which is not supported by developer btw)

You can connect circleci to your repo and change ci file [like this](https://github.com/Lastorder-DC/MusicBotKR/commit/9c6ce43cd219a6c0a3ba60c131c7ea6be02e3ce6) to download built jar file.

**Reply by GuardedBroD** [reply]
Cool! Thanks
